apidoc/Titanium/App/iOS/iOS.yml

```diff
@@ -1,1256 +1,1260 @@
 ---
 name: Titanium.App.iOS
 summary: |
-    The top-level App iOS module, available only to iOS devices, that includes the facilities to
-    create and manage local notifications and background services.
+The top-level App iOS module, available only to iOS devices, that includes the facilities to
```

Comment:
now this is even worse :-) Let's do that after merging then, just restore the file.